src/arena/SprintCreation.js

```diff
@@ -327,19 +328,21 @@ function SprintCreation(props) {
           </div>
         </div>
       ))}
-      <Calendar
-        onChange={(value) => {
-          setStartDate(value);
-          setReady(true);
-        }}
-        value={startDate}
-        maxDetail="month"
-        minDetail="month"
-        // minDate={new Date()}
-        maxDate={new Date(Date.now() + 2592000000)}
-        // tileDisabled={({ date }) => date.getDay() !== 1}
-        showNeighboringMonth
-      />
+      <LocalizationProvider dateAdapter={AdapterDateFns}>
```

Comment:
This is great! In terms of bundle size, it is around the same the `react-calendar` was, around 50kb but the theming should be much easier to fix.

Reply:
Can confirm the functionality seems to be working, great work!